fastlane/Fastfile

```diff
@@ -0,0 +1,20 @@
+default_platform(:ios)
+
+platform :ios do
+  desc "Build SwiftLeeds scheme"
+  lane :build do |options|
+    build_app(
+      project: "SwiftLeeds.xcodeproj",
+      configuration: options[:configuration],
+      scheme: "SwiftLeeds",
+      clean: true,
+      skip_archive: true,
+      skip_codesigning: true
+    )
+  end
+  
+  desc "Build SwiftLeeds scheme with Debug Configuration"
+  lane :build_debug do
```

Comment:
I tend to find convenience lanes like this can be quite useful when you've got parameterised lanes like above, helps remove some floaty strings

Reply:
I think yes, as we should prefer using specific lanes for each action where we need to call a parameterised lane.
Or do you see any better option to use `build` lane?

Reply:
I just assumed you'd use :build but pass in `configuration:Debug`, but happy either way

Reply:
I would prefer leaving all calls inside `Fastfile` and only call lanes from CI Providers in our case it's Github Actions. This will give us more stable infrastructure for future changes if any (Adding more lanes or Changing CI provider like Bitrise) so we only need to call lanes.